CODE_OF_CONDUCT.md

```diff
@@ -60,7 +60,7 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-florian.rupprecht@childmind.org.
+dair@childmind.org.
```

Comment:
Do you have access to this?

Reply:
Also, would it be prudent to make this a MOBI email?

Reply:
I asked John the other day, I'll ping him again to try and get more direct feedback. But overall he said it looked great

Reply:
I don't have access to that email. There is a generic MOBI one, but I put the dair one because this falls under Greg's umbrella? haha not sure